waisntechtools/alerts/Twilio/fetch_sms.py

```diff
@@ -0,0 +1,29 @@
+from twilio.rest import Client
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from twilio.request_validator import RequestValidator
+
+import requests
+import os
+
+
+def fetch_twilio(request):
+    account_sid = os.environ.get('TWILIO_ACCOUNT_SID')
+    auth_token = os.environ.get('TWILIO_AUTH_TOKEN')
+
+    validator = RequestValidator(auth_token)
+    request_valid = validator.validate(
+        request.build_absolute_uri(),
+        request.POST,
+        request.META.get('HTTP_X_TWILIO_SIGNATURE', ''))
+
+    if request_valid or settings.DEBUG:
+        client = Client(account_sid, auth_token)
```

Comment:
Is there a way where we could build the client once during service start up and cache it? I don't know how expensive this is, but it might make sense to cache the client.

Reply:
It is not something I considered.  My first thought is if this could be a security issue if we did it that way but I would need to look more into it